### ISSUE TITLE
Create a dockerfile

### ISSUE THREAD

**Issue by whispy** [curiositry/ReaDB#1]
It'd be nice if we could install this as a Docker container.

**Comment by curiositry** [comment]
Good idea @whispy! I’m not that familiar with Docker, but I’ll look into it as soon as I get a chance.

I’d also welcome pull requests for this.
